src/remote.ts

```diff
@@ -6,7 +6,9 @@ import * as fs from "fs/promises";
 import * as jsonc from "jsonc-parser";
 import * as os from "os";
 import * as path from "path";
-import prettyBytes from "pretty-bytes";
+// Dynamic import for ESM module
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+let prettyBytes: any;
```

Comment:
I know you said this was temporary, just leaving a note here to acknowledge we need to update this and other references before merge.  :smile: